rocky/OOI_database_seed.json

```diff
@@ -509,6 +509,19 @@
       "consult_api": false
     }
   },
+  {
+    "model": "tools.ooiinformation",
+    "pk": "KATFindingType|KAT-OPEN-COMMON-PORT",
+    "fields": {
+      "last_updated": "2021-08-09T14:50:09.835Z",
```

Comment:
Why do we have a timestamp here if it doesn't actually reflect when the record was updated?

Reply:
The last_updated field isn't used anywhere just like the consult_api field, I think we should just delete the fields. We should probably just delete those fields...

Reply:
isn't consult api used to check for updates when dealing with cve's etc? Those do need periodic updates.

Reply:
But it isn't used. The value is False by default, the value is False for everything in rocky/OOI_database_seed.json and there is no code that ever sets it to True as far as I can see.

Reply:
Yes true :). The idea was that when cve data was updated, people could use this to consult the cve database again (and other ooi information sources for that matter)